packages/waku/src/lib/middleware/types.ts

```diff
@@ -1,10 +1,13 @@
+import type { ModuleNode } from 'vite';
 import type { Config } from '../../config.js';
 import type { EntriesPrd } from '../../server.js';
 import type {
   renderRscWithWorker,
   getSsrConfigWithWorker,
 } from '../renderers/dev-worker-api.js';
 
+export type ClonableModuleNode = Pick<ModuleNode, 'url' | 'file'>;
```

Comment:
Can you define type simply without Pick? Seems like the values are just strings.